src/main/java/co/cask/gcp/common/GCPUtils.java

```diff
@@ -48,4 +49,25 @@ public static String getProjectId(@Nullable String project) {
     }
     return projectId;
   }
+
+  /**
+   *
```

Comment:
Missing java doc?

Reply:
It would be good to first merge release/0.11 to develop and then make these changes.